src/balena-event-log.js

```diff
@@ -231,7 +241,12 @@ module.exports = function (options) {
 				.then(function () {
 					return runAfterHook();
 				})
-				.asCallback(callback);
+				if (callback) {
+					p.then((result) => {
+						callback(null, result);
+					}, callback);
+				}
+				return p;
```

Comment:
Does balena-lint format JS? I guess not, because these lines seem to have the indentation problems.

Reply:
It does format javascript @roman-mazur, at least if you either explicitly specify the file or you use `-e js`